RevenueCatUI/Templates/V2/ViewModelHelpers/ViewModelFactory.swift

```diff
@@ -90,10 +90,22 @@ struct ViewModelFactory {
                 )
             }
 
+            let badgeViewModels = try component.badge?.stack.value.components.map { component in
```

Comment:
Deserialising the badge view models here just like we do with the stack children models right above.